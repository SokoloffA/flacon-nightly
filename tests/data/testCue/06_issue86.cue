REM GENRE Pop
REM DATE 1900
REM DISCID 12345678
REM COMMENT "ExactAudioCopy v0.99pb4"
PERFORMER "Performer Arts"
TITLE "Album"
FILE "Performer Arts - Title.wav" WAVE
  TRACK 01 AUDIO
    TITLE "Song"
    PERFORMER "Performer Arts"
    INDEX 00 00:00:00
    INDEX 01 00:00:32
  TRACK 02 AUDIO
    TITLE "Song"
    PERFORMER "Performer Arts"
    INDEX 00 03:58:25
    INDEX 01 03:59:10
  TRACK 03 AUDIO
    TITLE "Song"
    PERFORMER "Performer Arts"
    INDEX 00 08:15:02
    INDEX 01 08:15:22
  TRACK 04 AUDIO
    TITLE "Song"
    PERFORMER "Performer Arts"
    INDEX 00 11:37:52
    INDEX 01 11:37:57
  TRACK 05 AUDIO
    TITLE "Song"
    PERFORMER "Performer Arts"
    INDEX 00 14:57:17
    INDEX 01 14:57:18
  TRACK 06 AUDIO
    TITLE "Song"
    PERFORMER "Performer Arts"
    INDEX 00 18:51:70
    INDEX 01 18:51:73
    INDEX 02 22:50:54
  TRACK 07 AUDIO
    TITLE "Song"
    PERFORMER "Performer Arts"
    INDEX 00 22:49:67
    INDEX 01 22:52:00
  TRACK 08 AUDIO
    TITLE "Song"
    PERFORMER "Performer Arts"
    INDEX 00 26:23:25
    INDEX 01 26:23:50
  TRACK 09 AUDIO
    TITLE "Song"
    PERFORMER "Performer Arts"
    INDEX 00 30:39:27
    INDEX 01 30:40:62
  TRACK 10 AUDIO
    TITLE "Song"
    PERFORMER "Performer Arts"
    INDEX 00 34:28:27
    INDEX 01 34:28:42
  TRACK 11 AUDIO
    TITLE "Song"
    PERFORMER "Performer Arts"
    INDEX 00 37:00:67
    INDEX 01 37:01:72
  TRACK 12 AUDIO
    TITLE "Song"
    PERFORMER "Performer Arts"
    INDEX 00 41:35:00
    INDEX 01 41:35:10
