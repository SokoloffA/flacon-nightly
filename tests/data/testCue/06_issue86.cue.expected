[ DISC 01 / TRACK 01 ]
    INDEX CD 00 = 00:00:00
    INDEX CD 01 = 00:00:32

[ DISC 01 / TRACK 02 ]
    INDEX CD 00 = 03:58:25
    INDEX CD 01 = 03:59:10

[ DISC 01 / TRACK 03 ]
    INDEX CD 00 = 08:15:02
    INDEX CD 01 = 08:15:22

[ DISC 01 / TRACK 04 ]
    INDEX CD 00 = 11:37:52
    INDEX CD 01 = 11:37:57

[ DISC 01 / TRACK 05 ]
    INDEX CD 00 = 14:57:17
    INDEX CD 01 = 14:57:18

[ DISC 01 / TRACK 06 ]
    INDEX CD 00 = 18:51:70
    INDEX CD 01 = 18:51:73

[ DISC 01 / TRACK 07 ]
    INDEX CD 00 = 22:49:67
    INDEX CD 01 = 22:52:00

[ DISC 01 / TRACK 08 ]
    INDEX CD 00 = 26:23:25
    INDEX CD 01 = 26:23:50

[ DISC 01 / TRACK 09 ]
    INDEX CD 00 = 30:39:27
    INDEX CD 01 = 30:40:62

[ DISC 01 / TRACK 10 ]
    INDEX CD 00 = 34:28:27
    INDEX CD 01 = 34:28:42

[ DISC 01 / TRACK 11 ]
    INDEX CD 00 = 37:00:67
    INDEX CD 01 = 37:01:72

[ DISC 01 / TRACK 12 ]
    INDEX CD 00 = 41:35:00
    INDEX CD 01 = 41:35:10
